tests/source/comment_trailing_lines.rs

```diff
@@ -0,0 +1,21 @@
+// rustfmt-wrap_comments: true
+
+//! Module comment with multiple trailing lines
+//!
+//!
+
+/// Doc comment with multiple trailing lines
+///
+///
+pub mod foo {}
+
+/// Doc comment with one trailing line
+///
+pub mod bar {}
+
+/*
+ * Block comment with multiple trailing lines
+ *
+ *
+ */
+pub mod block {}
```

Comment:
Let's also add a test case like this:

```rust
/**
Outer Block doc comment with multiple trailing lines


 */
pub mod outer_block_doc_comment {}
```

Reply:
Is this the desired output of the above?
```rust
/**
Outer Block doc comment with multiple trailing lines
 */
pub mod outer_block_doc_comment {}
```

This case is more complex than the others, because block comments with bare lines go down a different code path unless `normalize_comments` is enabled (which rewrites the comment style altogether):
https://github.com/rust-lang/rustfmt/blob/ea02de27ff4119615c94107c6acb403b59593dae/src/comment.rs#L348-L349

To get the trailing blank line removal logic to work on this kind of comment, the above condition would have to be changed, which seems like it would cause other unexpected formatting?